<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wazon</class>
 <widget class="QWidget" name="Wazon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1690</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1690</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QStackedWidget" name="OrderBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="ProductTable">
           <property name="minimumSize">
            <size>
             <width>800</width>
             <height>0</height>
            </size>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
           </property>
           <column>
            <property name="text">
             <string>Клиент</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Цена</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Вес</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Количество</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Размер</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Платформа</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Артикул</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Срок доставки</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTableWidget" name="HistoryTable">
           <property name="minimumSize">
            <size>
             <width>900</width>
             <height>0</height>
            </size>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="ClientBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_3">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableWidget" name="ClientTable">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>ФИО</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Номер
 телефона</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Ближайший
 ПВЗ</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Адрес</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Сумма
 покупок</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Вес 
покупок</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Объем
 покупок</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_4">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTableWidget" name="CourierTable">
           <property name="editTriggers">
            <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
           </property>
           <column>
            <property name="text">
             <string>ФИО</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Номер
 телефона</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Транспортное 
средство</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Номер
 машины</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Грузоподъемность</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ActivOrd_History_btn">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>История</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Client_Courier_btn">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Курьеры</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="OverdueBox">
       <property name="text">
        <string>Просроченные</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_1">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="NewClient_btn">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Новый
клиент</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NewCourier_btn">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Новый
курьер</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NewOrder_btn">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Создать
заказ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NewRoute_btn">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Создать
маршрут</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
